codes/c/include/array_hash_map.h

```diff
@@ -0,0 +1,74 @@
+/**
+ * File: array_hash_map.h
+ * Created Time: 2023-03-18
+ * Author: Guanngxu (446678850@qq.com)
+ */
+
+#ifndef _HASH_MAP_H_
+#define _HASH_MAP_H_
+
+#include <stdio.h> 
+#include <stdlib.h> 
+#include <string.h> 
+#include <unistd.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* 哈希表默认数组大小 */
+# define HASH_MAP_DEFAULT_SIZE 100
+
+/* 键值对 int->string */
+typedef struct  {
+    int key;
+    char *val;
+} entry_t;
+
+/* 基于数组简易实现的哈希表 */
+typedef struct {
+    // 初始化数组，包含 100 个桶
+    entry_t buckets[HASH_MAP_DEFAULT_SIZE]; /** Entries in hash_map */
```

Comment:
The pointer to `entry_t` should be stored here, such as
`entry_t *buckets[HASH_MAP_DEFAULT_SIZE];`

The purpose of this is to try to maintain consistency with other languages